static/src/javascripts/projects/commercial/modules/article-aside-adverts.ts

```diff
@@ -0,0 +1,109 @@
+import { $$ } from '../../../lib/$$';
+import config_ from '../../../lib/config';
+import fastdom from '../../../lib/fastdom-promise';
+import mediator from '../../../lib/mediator';
+
+const minArticleHeight = 1300;
+
+const config = config_ as {
+	get: (s: string, b?: boolean) => string;
+};
+
+const getAllowedSizesForImmersive = (availableSpace: number) => {
+	// filter ad slot sizes based on the available height
+	// mark: 01303e88-ef1f-462d-9b6e-242419435cec
```

Comment:
This usually indicates that there's a equivalent file elsewhere (in this case DCR):

https://github.com/search?q=org%3Aguardian+01303e88-ef1f-462d-9b6e-242419435cec&type=code